.travis.yml

```diff
@@ -44,12 +44,16 @@ jobs:
       env:
         - TEST_NAME=COVERAGE
       before_install:
+        - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
```

Comment:
I think we already have `ubuntu-toolchain-r-test` in sources. Please look at line 9